main.go

```diff
@@ -443,8 +443,8 @@ The output directory is not empty. Please choose one of actions below:
 	app := &cli.App{
 		Name:      "aztfy",
 		Version:   getVersion(),
-		Usage:     "Bring existing Azure resources under Terraform's management",
-		UsageText: "aztfy [command] [option]",
+		Usage:     "A tool to help bring existing Azure resources under Terraform's management",
```

Comment:
```suggestion
		Usage:     "A tool to bring existing Azure resources under Terraform's management",
```

This makes it consistent with the repo's description.